Allow disabling congestion control for IP flows (#162)
ietf-wg-masque/draft-ietf-masque-connect-ip: draft-ietf-masque-connect-ip.md

{{TCP}} or {{QUIC}}), the proxied traffic will incur at least two nested
congestion controllers. When tunneled packets are sent using QUIC DATAGRAM
frames, the outer HTTP connection MAY disable congestion control for those

> packets that contains QUIC DATAGRAM frames encapsulating IP packets and no other
> DATA frames. Implementers will benefit from reading the guidance in {{Section

Reviewer: ```suggestion
packets that contain only QUIC DATAGRAM frames encapsulating IP packets.
Implementers will benefit from reading the guidance in {{Section
```
